rct229/rulesets/ashrae9012019/section11/section11rule6.py

```diff
@@ -0,0 +1,97 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
```

Comment:
Weili's comment:

My take is this rule should be at ```service_water_heating_distribution_systems``` level, not building level. (See RDS Evaluation Context)

In RMD Rule, we need to add an ```is_applicable``` to check if there are ```service_water_heating_distribution_systems``` or not.
In RMD Rule ```create_data```, we should call ```get_swh_components_associated_with_each_swh_distribution_system``` to get the data.

In the ```ServiceWaterHeatingDistributionSystem```, we need to check every piping, including children, is modeled with thermal losses turned to false.
You can reuse the same code block searches for piping under a ```ServiceWaterHeatingDistributionSystem``` from the ```get_swh_components_associated_with_each_swh_distribution_system``` function.